ansible-ipi-install/roles/node-prep/tasks/08_create_config_install_dirs.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: IPI-INSTALL Create config dir and install dir
```

Comment:
this seems more related to the installer steps instead of node-prep

Reply:
I can see that, but I'm just creating directories. You can make the argument, I'm "prepping" so that the configs and install binaries can be set in place.